cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -222,7 +222,8 @@ class AnalyzerCommand : OrtCommand(
         )
 
         val curationCount = packages.sumOf { it.curations.size }
-        println("Applied $curationCount curation(s) from ${enabledCurationProviders.size} provider(s).")
+        val providerCount = ortResult.resolvedConfiguration.packageCurations.count { it.curations.isNotEmpty() }
+        println("Applied $curationCount curation(s) from $providerCount provider(s).")
```

Comment:
I'm not sure about this change, I think it could be confusing if you configure e.g. 3 providers and the log says "Applied x curation(s) from 1 provider(s)." Or does it print the used providers somewhere next to it?

Reply:
> Or does it print the used providers somewhere next to it?

It does print them above in line 171. Having the difference about the number of enabled providers vs. how many of them actually contributed curations is exactly the stats I'm interested in.

Reply:
Then how about printing the number of curations for each provider here, wouldn't that be even better?

Reply:
It would add more detail, yes, but I didn't want to print full-blown stats, but only a short summary that always fits into a single line.

Reply:
Ok, could you change it to print "from x of y (or x/y) provider(s)" then? This would solve the issue for me.